src/storage/heap_file.c

```diff
@@ -21641,6 +21786,18 @@ heap_update_bigone (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context, b
 	  ASSERT_ERROR_AND_SET (error_code);
 	  goto exit;
 	}
+
+/*SUPPLEMENT_UPDATE REDO LSA (mvcc), UNDO,REDO (non-mvcc) bigone to bigone */
+      if (is_supplement)
+	{
+	  LSA_COPY (&redo_lsa, &tdes->tail_lsa);
+
+	  if (!is_mvcc_op)
+	    {
+	      LSA_COPY (&undo_lsa, &tdes->tail_lsa);
```

Comment:
same lsa to redo_lsa. You're going to traverse from the same lsa?

Reply:
Yes. for non-mvcc update (bigone to bigone), I have to traverse from same lsa. because the logs are appended inside the heap_ovf_update(). Traverse from the lsa, and find the REDO or UNDO log records until it meets the LOG_DUMMY_OVF_RECORD.